<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QWidget" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>591</width>
     <height>451</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>   登录   </string>
    </attribute>
    <widget class="QLabel" name="lb_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_login">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>登录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>   注册   </string>
    </attribute>
    <widget class="QPushButton" name="pb_clear">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>清空</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_register_name">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lb_register_name">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>用户名</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_register_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>180</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>密码</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_register_password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>180</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_register_confirm_password">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>260</y>
       <width>311</width>
       <height>31</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="lb_register_confirm_password">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>确认密码</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_register">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>330</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>注册</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>le_name</tabstop>
  <tabstop>le_password</tabstop>
  <tabstop>pb_login</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>le_register_name</tabstop>
  <tabstop>le_register_password</tabstop>
  <tabstop>le_register_confirm_password</tabstop>
  <tabstop>pb_register</tabstop>
  <tabstop>pb_clear</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
